test/functional/feature_rbf.py

```diff
@@ -13,10 +13,12 @@
 
 MAX_REPLACEMENT_LIMIT = 100
 
+DUMMY_P2WPKH_SCRIPT = CScript([bytes([1]) * 21])
```

Comment:
to be clear: P2WPKH scripts are 22 length scripts: OP_0 <PUSH 20> <20-bytes>

Reply:
Yes, the 21 is chosen by purpose, as the constructor seems to add an implicit push instruction here, resulting in the desired 22 bytes -- printing length and content of `DUMMY_P2WPKH_SCRIPT` gives:
```
22

b'\x15\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01'
``` 
Should I maybe add a comment noting that the result is 22 bytes?

Reply:
I'd be very surprised that the leading byte isn't the serialize byte for the entire script

Reply:
As far as I can see the length of a scriptPubKey is serialized in class TxOut (respectively for scriptSig in class TxIn) rather directly in the class CScript itself, see https://github.com/bitcoin/bitcoin/blob/master/test/functional/test_framework/messages.py#L337 and https://github.com/bitcoin/bitcoin/blob/master/test/functional/test_framework/messages.py#L89.

Quick check in the python console:
```
>>> from test_framework.messages import *
>>> from test_framework.script import *
>>> txn = CTransaction()
>>> txn.vin.append(CTxIn())
>>> txn.vout.append(CTxOut(0, CScript([b'a' * 21])))
>>> len(txn.serialize())
82
```

Reply:
```
+        # Transaction size
+        txn = CTransaction()
+        txn.vin.append(CTxIn())
+        txn.vout.append(CTxOut(0, b''))
+        print("txn size: " +str(len(txn.serialize())))
+        return
------
txn size: 60
```
:thinking:

Reply:
```
        # Transaction size
        txn = CTransaction()
        txn.vin.append(CTxIn())
        txn.vout.append(CTxOut(0, b'a'*21))
        print("txn size: " +str(len(txn.serialize())))
        return
-----
txn size: 81
```

Reply:
ah! I'm sorry you're using a different constructor, continue :)

Reply:
I actually tried the simpler "pure" construction of the script with just e.g. `b'a' * 22` but then the tests would fail because the script contents don't make sense:
`test_framework.authproxy.JSONRPCException: non-mandatory-script-verify-flag (Data push larger than necessary) (code 64) (-26)`
or
`test_framework.authproxy.JSONRPCException: mandatory-script-verify-flag-failed (Script evaluated without error but finished with a false/empty top stack element) (code 16) (-26)`

Another possibility would be to explicitely add the push, like 
`DUMMY_P2WPKH_SCRIPT = bytes([21]) + b'a' * 21`

Reply:
right it's not even a parseable script